packages/design-system/src/components/Banner/Banner.test.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import Banner from './Banner';
+
+describe('Banner', () => {
+  it('should render small info Banner', () => {
```

Comment:
well, it says `should render small info Banner`, but it's actually `should render banner of any type and size`